gfx/UltimateCataclysm/pngs_incomplete_32x32/items/candle/credits.txt

```diff
@@ -1,3 +0,0 @@
-https://sketchfab.com/3d-models/candle-light-animated-d5e7042d18a14127a9971bb493b8c863
-model by CATENCODE
-rendered by int_ua
```

Comment:
I think the new candle is too similar to completely remove `credits.txt`. I think you should add yourself to the file instead of removing it.

Reply:
Done